<ui version="4.0" >
 <class>SolutionView</class>
 <widget class="QWidget" name="SolutionView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Solution View</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="moveButton" >
       <property name="text" >
        <string>Move</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="autoExclusive" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomButton" >
       <property name="text" >
        <string>Zoom</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="autoExclusive" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="rotateButton" >
       <property name="text" >
        <string>Rotate</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="autoExclusive" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionZoom" >
   <property name="text" >
    <string>zoom</string>
   </property>
  </action>
  <action name="actionMove" >
   <property name="text" >
    <string>move</string>
   </property>
  </action>
  <action name="actionRotate" >
   <property name="text" >
    <string>rotate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
